File: src/main/java/lab3/Simulation.java
package lab3;

import lab3.car_bank.CarBankModel;

public class Simulation {
    public static void main(String[] args) {

//        Process first = new Process(4, 2, () -> FunRand.Exp(0.3));
//        Process second = new Process(4, 2, () -> FunRand.Exp(0.3));
//        Create generator = new Create(0.1, () -> FunRand.Exp(0.5));

        CarBankModel model = new CarBankModel();

        model.simulate(100);
    }
}

File: src/main/java/lab3/clinic/Simulation.kt
package lab3.clinic

import lab2.simsimple.FunRand
import lab3.entity.Create
import lab3.entity.Process
import java.util.function.Function

fun main() {

    val delayGenerator: Function<Int, Double> = Function { FunRand.Exp(15.0) }
    val router: Function<MutableList<Process>, Process> = Function { it.first() }

//    val create = Create(
//        0.0,
//        delayGenerator,
//        mutableListOf<Process>(),
//        router
//    )
}